model/Role/Role.php

```diff
@@ -31,13 +31,13 @@ class Role extends \DbObject
     public const CLEN_RADY            = 23;
     public const SEF_INFOPULTU        = 24;
     public const SEF_PROGRAMU         = 25;
+    public const MINI_ORG             = 26;
```

Comment:
Jelikož to chtějí jako ročníkovou roli, tak tohle musíme předělat na dynamickou konstantu (zní to divně :-)

Po vzoru `\Gamecon\Role\Role::LETOSNI_VYPRAVEC`